server/lib/data_import/my_anime_list.rb

```diff
@@ -0,0 +1,43 @@
+module DataImport
+  class MyAnimeList
+    MDL_HOST = "https://hbv3-mal-api.herokuapp.com/2.1" # 2.1 is the api version, MAYBE add graceful decay to check from v1 if it can't find?
+
+    include DataImport::Media
+    include DataImport::HTTP
+
+    attr_reader :opts
+
+    def initialize(opts = {})
+      @opts = opts.with_indifferent_access
+      super()
+    end
+
+    def get_media(external_id) # anime/1234 or manga/1234
+      media = Mapping.lookup('myanimelist/anime', external_id) || Anime.new # not sure if the lookup is correct
```

Comment:
I don't remember which we actually _have_ right now, but I'm planning to ensure it's `site: 'myanimelist', id: 'anime/1234'` for various reasons (one of which being how easy it makes URL-building!)

So probably wanna drop the `/anime` in the lookup call.  As for the `Anime.new` call, you can probably do `external_id.split('/').first.classify.constantize` instead to find the class ;)